test/integration/verify/controls/azure_power_bi_dashboard.rb

```diff
@@ -0,0 +1,10 @@
+group_id = input(:inspec_powerbi_workspace_id, value: '')
+dashboard_id = 'b84b01c6-3262-4671-bdc8-ff99becf2a0b'
```

Comment:
can not keep the dashboard id hardcoded I think or need to provide through variable file if it does not change.